doc/source/whatsnew/v0.15.2.txt

```diff
@@ -163,7 +163,7 @@ Other enhancements:
      p.all()
 
 - Added support for ``utcfromtimestamp()``, ``fromtimestamp()``, and ``combine()`` on `Timestamp` class (:issue:`5351`).
-- Added Google Analytics (`pandas.io.ga`) basic documentation (:issue:`8835`). See :ref:`here<remote_data.ga>`.
+- Added Google Analytics (`pandas.io.ga`) basic documentation (:issue:`8835`). See `here<http://pandas.pydata.org/pandas-docs/version/0.15.2/remote_data.html#remote-data-ga>`__.
```

Comment:
Hmm, I don't want to start updating whatsnew files if we change things (there are other mentions of the ga moudels in whatsnew files as well). In principle no one should read this now anymore .. 
I just wanted to make sure to not have a doc build warning about a non-existing label, could remove the link altogether as well.